RELEASE.md

```diff
@@ -0,0 +1,44 @@
+# Team
+
+The release team is made up of the following:
+
+* Rotating release engineer from one of the development teams
+* SRE team supporting deployment and CI/CD
+* Test Engineers that focus on integration tests
+* Performance Engineers running benchmarking tests
+
+# Release Process
+
+This is the release process for node releases
+
+1. Release Squad Lead will work with rotating engineer and dev leads to determine where to cut the releases from ledger, network and node. These will be published via CHaP and will follow the defined process of the team for versioning, etc...
+2. These will be integrated up the stack to the node to produce a release branch
```

Comment:
Shouldn't this be done at `main` branch level? eg. update dependencies in discrete step, run integration tests, if it's green and we're happy make it a release branch (or tag)?

Reply:
but we are going to integrate those changes through versions, right? eg. bump `ouroboros-consensus` package version to `1.2.3`?